src/wazuh_modules/wm_ciscat.c

```diff
@@ -116,31 +116,36 @@ void* wm_ciscat_main(wm_ciscat *ciscat) {
     // Define path where CIS-CAT is installed
 
     if (ciscat->ciscat_path) {
-        char pwd[PATH_MAX];
-
         switch (wm_relative_path(ciscat->ciscat_path)) {
-            case 0:
+            case 0: {
                 // Full path
                 snprintf(cis_path, OS_MAXSTR - 1, "%s", ciscat->ciscat_path);
-                break;
-            case 1:
+            }
+            break;
+
+            case 1: {
                 // Relative path
             #ifdef WIN32
                 if (*current) {
                     snprintf(cis_path, OS_MAXSTR - 1, "%s\\%s", current, ciscat->ciscat_path);
                 }
             #else
+                char pwd[PATH_MAX];
+
                 if (getcwd(pwd, sizeof(pwd)) == NULL) {
                     mterror(WM_CISCAT_LOGTAG, "Could not get the current working directory: %s (%d)", strerror(errno), errno);
                     ciscat->flags.error = 1;
                 } else {
                     os_snprintf(cis_path, OS_MAXSTR - 1, "%s/%s", pwd, ciscat->ciscat_path);
                 }
             #endif
-                break;
-            default:
+            }
+            break;
+
+            default: {
                 mterror(WM_CISCAT_LOGTAG, "Defined CIS-CAT path is not valid.");
                 ciscat->flags.error = 1;
+            }
```

Comment:
Case blocks only need scope for case 1 when pwd is stack-allocated. Other ones does not need it.
Also break statements  and closing brance should bot be aligned with case statements. See https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C%29#style-conventions